components/automate-ui/e2e/admin.e2e-spec.ts

```diff
@@ -118,7 +118,7 @@ describe('Admin pages', () => {
           {
             policies: [
               {
-                id: 'some-policy-id',
+                id: 'Custom',
```

Comment:
Please leave theses as-is. The "Custom" and "Chef-managed" are the **types** of the policies, as displayed in the UI, and as asserted by the the tests.